packages/patternfly-4/react-charts/src/components/ChartTheme/themes/theme-victory.js

```diff
@@ -0,0 +1,235 @@
+/* eslint-disable camelcase */
+import { global_FontFamily_sans_serif } from '@patternfly/react-tokens';
+
+// TODO Replace values with PF css variable when available
+
+// Typography
+const TYPOGRAPHY_FONT_FAMILY = global_FontFamily_sans_serif.value;
+const TYPOGRAPHY_LETTER_SPACING = 'normal';
+const TYPOGRAPHY_FONT_SIZE = 14; // Value must be in pixles
+
+// Labels
+const LABEL_PROPS = {
+  fontFamily: TYPOGRAPHY_FONT_FAMILY,
+  fontSize: TYPOGRAPHY_FONT_SIZE,
+  letterSpacing: TYPOGRAPHY_LETTER_SPACING,
+  padding: 10, // Value must be in pixles
+  stroke: 'transparent'
+};
+const LABEL_CENTERED_PROPS = {
+  ...LABEL_PROPS,
+  textAnchor: 'middle'
+};
+
+// Layout
+const LAYOUT_PROPS = {
```

Comment:
Remove?

Reply:
I wasn't ready to remove that quite, yet. Wanted to see what variables are created in patternfly core

Reply:
Could we just add it back in when we pull in PF variables? It's just dead code that will be merged into master.

Reply:
I want to leave it for now because I'm not finished working on with the theme. If we end up with no layout properties (e.g., a default height and width), I will remove it.